test/unit/spec/ForgotPassword_spec.js

```diff
@@ -228,7 +228,7 @@ function (Q, Okta, OktaAuth, Util, AccountRecoveryForm, PrimaryAuthForm, Beacon,
           test.form.sendEmail();
           expect($.ajax.calls.count()).toBe(0);
           expect(test.form.usernameErrorField().length).toBe(1);
-          expect(test.form.usernameErrorField().text()).toBe('The field cannot be left blank');
+          expect(test.form.usernameErrorField().text()).toBe('This field cannot be left blank');
```

Comment:
fix: unit test assertion

Reply:
do we need to fix this test too ? https://travis-ci.org/okta/okta-signin-widget/builds/480596228#L2319

Reply:
yes @nikhilvenkatraman-okta I just updated it. Missed it earlier.